docs/community/glossary.md

```diff
@@ -0,0 +1,15 @@
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| channel            | A channel is a grayscale image. In fluorescence microscopy,  each channel corresponds to a range of wavelengths. An RGB image from digital camera consists of 3 channels - red, green, blue.                                                                                                                                                |
+| color map          | An array of colors used to map pixel data to the actual color values.                                                                                                                                                                                                                                                                       |
+| dock widget        | A widget that can be attached to the main napari viewer at a specified position.                                                                                                                                                                                                                                                            |
+| hook               | Hook is the place within napari where we “promise” to call functions created by external developers and installed by the user.                                                                                                                                                                                                              |
+| hook specification | A function signature that defines the API that a plugin developer must adhere to when writing the function that we promise to call somewhere in the napari codebase.                                                                                                                                                                        |
+| keybinding         | The key or key combination that causes some functions to occur.                                                                                                                                                                                                                                                                             |
+| labels             | The labels layer is an image where pixel values are integers. Each unique integer represents a unique object, feature or classification.                                                                                                                                                                                                    |
```

Comment:
Definition seems reasonable, should probably link to the docs page for labels layers.

Reply:
I wonder if labels being defined directly as the labels layer is a bit confusing. Maybe:

"A label is a region of an integer image where the pixels are the same unique value. They are implemented in napari in the labels layer where each label represents a unique object, feature or classification"

I think these are also known as ROIs? Not sure if that is a correct equivalence...

Reply:
Thanks @DragaDoncila I will update labels definition. 
I don't think it's equivalent to ROI though. Yes we can use labels to define ROI, but that can also be achieved by shapes or other means.

Reply:
> I don't think it's equivalent to ROI though. Yes we can use labels to define ROI, but that can also be achieved by shapes or other means.

Agreed, these aren't quite the same